.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+ports:
+  - port: 8080
+  - port: 3000
```

Comment:
As I mentioned before I think frontend contributors are more likely to be experienced with npm, etc., and can probably get a development environment going pretty quickly. So let's keep this focused on running the backend, and remove port 3000 for now.